tools/lib/utils.js

```diff
@@ -0,0 +1,36 @@
+const last = require('lodash/last')
+
+module.exports = { getPresetIds, formatItems }
+
+const presetCategories = {
+  base: null,
+  essential: 'base',
+  'vue3-essential': 'base',
+  'strongly-recommended': 'essential',
+  'vue3-strongly-recommended': 'vue3-essential',
+  recommended: 'strongly-recommended',
+  'vue3-recommended': 'vue3-strongly-recommended'
+  // 'use-with-caution': 'recommended',
+  // 'vue3-use-with-caution': 'vue3-recommended'
+}
+
+function formatItems(items) {
+  if (items.length <= 2) {
+    return items.join(' and ')
+  }
+  return `all of ${items.slice(0, -1).join(', ')} and ${last(items)}`
```

Comment:
With `[...items].pop()` or `items.at(-1)`, we could avoid the lodash dependency.